ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -94,7 +94,7 @@ protected CompletableFuture<FastSyncState> start(final FastSyncState fastSyncSta
           onBonsai.clearFlatDatabase();
           onBonsai.clearTrieLog();
         });
-    LOG.debug("Start sync with initial sync state {}", fastSyncState);
+    LOG.debug("Start fast sync with initial sync state {}", fastSyncState);
```

Comment:
maybe we can do the same as here https://github.com/hyperledger/besu/pull/7140/files#diff-c971b57a1d994b1c08fd5f43fd3cf282a41ccd524cafa8421c6eb1bad9087b08R83

Reply:
In this case it genuinely is `fast` sync. The `start()` method is common to this class and the `SnapSyncDownload` subclass.

The `start(final FastSyncState fastSyncState)` method is actually doing `fast` sync in this class, and is doing `snap` sync in `SnapSyncDownloader.java` so in `SnapSyncDownloader` the log says "Start snap sync..."

Reply:
Ok missed that thanks 🙏